pom.xml

```diff
@@ -186,5 +186,10 @@
             <artifactId>commons-lang3</artifactId>
```

Comment:
All these dependencies are not needed now as they will come with zecko-common-public

Reply:
understood